config/initializers/delayed_job_config.rb

```diff
@@ -0,0 +1,14 @@
+Delayed::Worker.destroy_failed_jobs = false
+Delayed::Worker.sleep_delay = 60
+Delayed::Worker.max_attempts = 3
+Delayed::Worker.max_run_time = 30.minutes
+Delayed::Worker.read_ahead = 10
+Delayed::Worker.default_queue_name = 'default'
+Delayed::Worker.delay_jobs = !Rails.env.test?
+Delayed::Worker.raise_signal_exceptions = :term
+
+Delayed::Worker.logger ||= if defined?(Rails)
```

Comment:
What happens if neither?

Reply:
this is fixed in https://github.com/openregister/registers-frontend/pull/116/commits/dc5fa4f17f1d9ba570e161ae02f5de6d6cce49db